### ISSUE TITLE
removed from Google play?

### ISSUE THREAD

**Issue by paultag** [guardianproject/lildebi#174]
Any reason why?

**Comment by eighthave** [comment]
Yes its true. We're working on bringing it back to Google Play, but it seems that more people got it from Fdroid than Google Play anyhow.

The Lil' Debi icon apparently looked too much like the Little Debbie cakes icon, and then Little Debbie cakes finally made a (pointless) app, so I guess they noticed, and complained to Google.  I have since redone the icon to make it look less like the icon of Little Debbie cakes. It seems possible to get Google to unsuspend the app but I have had no luck so far.  Anyone have advice on that process?
